### PR TITLE
Feat: remove channel name in schema and add channel id

### PR DESCRIPTION
**Pull request by magicpaIm**
[fix #24](https://github.com/RnDAO/tc-DAOlytics/issues/24)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,6 @@ All interactions with DB
 rawinfo {
     messageId: Snowflake,(unique)
     type?: string,
-    channel: string,
```
Comment (mehrdadmms) [inline README.md:11]: We need this, please replace with channel id

Comment (mehrdadmms) [reply README.md:11]: we need to know where the message is from

Comment (mehrdadmms) [reply README.md:11]: Please deploy this on dev env before moving to prod.

**Review by mehrdadmms — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (magicpaIm) [reply README.md:11]: > We need this, please replace with channel id
channelId is already in schema, just removed the channel field.

Comment (magicpaIm) [reply README.md:11]: > Please deploy this on dev env before moving to prod.

I don't know how to deploy on dev env

**Pull request merged**